testgrid/config.yaml

```diff
@@ -6625,6 +6625,17 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gke-regional-slow
     base_options: include-filter-by-regex=Volume%7Cstorage
     description: storage gke regional slow e2e tests for master branch
+  dashboard_names:
+  - sig-storage-local-static-provisioner
+
+- name: sig-storage-local-static-provisioner
+  dashboard_tab:
+  - name: master-gce-lastest
+    test_group_name: ci-sig-storage-local-static-provisioner-master-gce-latest
+    description: 'E2E tests for kubernetes latest version on GCE'
+  - name: master-gke-default
+    test_group_name: ci-sig-storage-local-static-provisioner-master-gke-default
+    description: 'E2E tests for kubernetes default version on GKE'
```

Comment:
I followed `sig-gcp-compute-persistent-disk-csi-driver` to add CI jobs in sub-dashboard under sig-storage.

https://github.com/kubernetes/test-infra/blob/f5ac6c3a74a392c00969aabaa458148b27ad9673/testgrid/config.yaml#L7739-L7744

https://github.com/kubernetes/test-infra/blob/f5ac6c3a74a392c00969aabaa458148b27ad9673/testgrid/config.yaml#L5934-L5938